Style/style.css

```diff
@@ -783,3 +773,204 @@ section.about-me-page .grid-skill-display .skill-card .skill-card-details .skill
   text-align: center;
   color: #42526e;
 }
+
+@media (min-width: 768px) {
+  .homepage-container {
+    background-image: url(../Assets/Images/homepage-background-desktop.svg);
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: 50% 50%;
+  }
+
+  .homepage-container .homepage-header {
+    display: none;
+  }
+
+  .homepage-container .homepage-header-large-screen {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-pack: center;
+    -ms-flex-pack: center;
+    justify-content: center;
+    padding: 8px;
+    margin-top: 10px;
+  }
+
```

Comment:
I am really impressed by your project at this stage :100: . Keep it up, bro. :+1: The only issue I have is that the design does not conform to the desktop figma design between screen width of 768px and 992px. Check the images below :point_down:  for illustration. Kindly ensure that it conforms to the dimensions of the desktop figma design for all screen size greater that 768px as stated in the project requirements. Also for your design template the project cards should be 6 instead of 5. 

**Headline Section**

Your design             |  Figma design
:-------------------------:|:-------------------------:
![Screenshot from 2022-09-30 12-51-53](https://user-images.githubusercontent.com/65233860/193277608-6f3671c2-bf4c-45d6-9d1d-1c42c433237f.png)  |  ![Screenshot from 2022-09-30 12-52-17](https://user-images.githubusercontent.com/65233860/193277647-c09515f9-d672-4e2f-9617-a46d386013a4.png)



**Project Section**

Your design             |  Figma design
:-------------------------:|:-------------------------:
![Screenshot from 2022-09-30 12-53-17](https://user-images.githubusercontent.com/65233860/193277773-8a3b495c-d62b-48a5-b90e-1a338fa4f5f7.png)  |  ![Screenshot from 2022-09-30 12-52-31](https://user-images.githubusercontent.com/65233860/193277800-d8eb1331-a7df-4f0c-90e4-bdbc31dfe5bc.png)



**About Me Section**

Your design             |  Figma design
:-------------------------:|:-------------------------:
![Screenshot from 2022-09-30 12-54-16](https://user-images.githubusercontent.com/65233860/193278045-b65ee68d-2f5a-45cf-b765-9e35be118552.png)  |  ![Screenshot from 2022-09-30 12-54-28](https://user-images.githubusercontent.com/65233860/193278078-8ab70649-3f2a-408c-882a-71317c32d944.png)




**Contact Section**

Your design             |  Figma design
:-------------------------:|:-------------------------:
![Screenshot from 2022-09-30 12-54-51](https://user-images.githubusercontent.com/65233860/193278178-225b4009-d638-4fa4-bea8-ad530af31566.png)  |  ![Screenshot from 2022-09-30 12-54-36](https://user-images.githubusercontent.com/65233860/193278195-1137572d-ffd9-4460-9b3f-6fca7646bc25.png)